modules/react/table/lib/Table.tsx

```diff
@@ -66,11 +174,11 @@ export default function App() {
  */
 export const Table = createComponent('table')({
   displayName: 'Table',
-  Component: ({children, ...elemProps}: GridProps, ref, Element) => {
+  Component: ({children, ...elemProps}: TableProps, ref, Element) => {
     return (
-      <Element ref={ref} {...mergeStyles(elemProps, tableStencil())}>
+      <BaseTable as={Element} ref={ref} {...mergeStyles(elemProps, tableStencil())}>
```

Comment:
I think Element is fine here after talking to Nicholas no?

Reply:
Yeah probably. I can update it.